client/a8c-for-agencies/sections/agency-tier/early-access-banner/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useTranslate } from 'i18n-calypso';
+import LayoutBanner from 'calypso/a8c-for-agencies/components/layout/banner';
+import { useDispatch, useSelector } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getPreference } from 'calypso/state/preferences/selectors';
+
+const EARLY_ACCESS_BANNER_DISMISSED = 'a4a-agency-tier-early-access-banner-dismissed';
+
+export default function EarlyAccessBanner() {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const bannerDismissed = useSelector( ( state ) =>
+		getPreference( state, EARLY_ACCESS_BANNER_DISMISSED )
+	);
+
+	const handleDismiss = () => {
+		dispatch( savePreference( EARLY_ACCESS_BANNER_DISMISSED, true ) );
+		dispatch( recordTracksEvent( 'calypso_a4a_early_access_banner_dismissed' ) );
```

Comment:
Do you think we should update the event name?

```suggestion
		dispatch( recordTracksEvent( 'calypso_a4a_agency_tier_early_access_banner_dismissed' ) );
```

Reply:
Thanks for the review, I've renamed it to `calypso_a4a_early_access_banner_dismiss_click`